src/optimization/lbfgs.cpp

```diff
@@ -137,20 +135,17 @@ void lbfgs_solver::setup(
     const DenseVector& init_point,
     const std::map<std::string, flexible_type>& opts,
     const std::shared_ptr<smooth_regularizer_interface>& _reg) {
-  
   m_status = solver_status();
   reg = _reg;
 
-  auto get_param = [&](const std::string& n) { 
+  auto get_param = [&](const std::string& n) {
     auto it = opts.find(n);
     return (it != opts.end()) ? it->second : default_solver_options.at(n);
   };
 
-
   lbfgs_memory_level = get_param("lbfgs_memory_level");
   convergence_threshold = get_param("convergence_threshold");
-  m_status.step_size = 1; // get_param("step_size");
-
+  m_status.step_size = 1.0;  // / std::sqrt(num_variables);
```

Comment:
What's with the comment? The value doesn't seem to match the comment...

Reply:
Bad comment.  It's set automatically now.